src/goto-programs/remove_exceptions.cpp

```diff
@@ -226,9 +232,13 @@ static goto_programt::targett get_exceptional_output(
     const irep_idt &statement=it->code.get_statement();
     if(statement==ID_output)
     {
-      assert(it->code.operands().size()>=2);
+      INVARIANT(
+        it->code.operands().size()>=2,
+        "Output expected to have at least 2 operands");
```

Comment:
Nitpicking on uniformity: Is there a guideline on whether the message starts lower or upper case?